core/fx/retry.go

```diff
@@ -1,31 +1,76 @@
 package fx
 
-import "github.com/zeromicro/go-zero/core/errorx"
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/zeromicro/go-zero/core/errorx"
+)
 
 const defaultRetryTimes = 3
 
+var timeoutErr = errors.New("retry timeout")
```

Comment:
Recommend to change to the form like `errTimeout`.

Reply:
Of course, it's been revised